src/plugins/irc/irc-server.c

```diff
@@ -3387,7 +3439,7 @@ irc_server_connect_cb (void *data, int status, int gnutls_rc, int sock,
             server->current_ip = (ip_address) ? strdup (ip_address) : NULL;
             weechat_printf (
                 server->buffer,
-                _("%s%s: connected to %s/%d (%s)"),
+                "%s%s: connected to %s/%d (%s) lolwat",
```

Comment:
That was from the early stages of debugging, when I was still trying to get my change/compile/test workflow working for WeeChat. Please ignore :).